dll/shellext/shellbtrfs/recv.cpp

```diff
@@ -1736,6 +1645,11 @@ void CALLBACK RecvSubvolW(HWND hwnd, HINSTANCE hinst, LPWSTR lpszCmdLine, int nC
 
             tp->PrivilegeCount = 3;
 
+#ifndef __REACTOS__
```

Comment:
The WinBtrfs author doesn't compile his code with MSVC too? (aka. would he have then noticed whether these pragmas are just ignored there, or would error out as being unrecognized?)

Reply:
I'll setup MSVC to test.
but WinMG on Linux is not happy about it.

Reply:
gcc was the one not happy due to the -Wall adding -Wunknown-pragmas
changed to `ifdef __clang__`.